docs/version0.33.md

```diff
@@ -35,10 +35,12 @@ The following new features and notable changes since version 0.30.0 are included
 
 ### Binaries and supported environments
 
-OpenJ9 release 0.33.0 supports OpenJDK 8, 11, 17, and 18.
+OpenJ9 release 0.33.0 and 0.33.1 supports OpenJDK 8, 11, 17, and 18.
```

Comment:
Should be "releases support" not "release supports"

Reply:
Used 0.33.x here as well.